src/components/Announcement.tsx

```diff
@@ -12,27 +12,36 @@ import classnames, {
   flex,
   fontSize,
   fontWeight,
+  height,
   justifyContent,
   lineHeight,
   margin,
+  opacity,
   padding,
+  transitionDuration,
   transitionProperty,
-  translate,
+  transitionTimingFunction,
 } from 'classnames/tailwind'
 import walletStore from 'stores/WalletStore'
 
+const commonTransition = classnames(
+  transitionDuration('duration-75'),
+  transitionTimingFunction('ease-linear')
+)
+
 const announceWrapper = (animate?: boolean) =>
   classnames(
     backgroundColor('bg-primary-dimmed'),
-    padding('px-6', 'py-4'),
+    padding(animate ? 'px-0' : 'px-6', animate ? 'py-0' : 'py-4'),
     display('flex'),
     alignItems('items-center'),
     justifyContent('justify-center'),
     fontWeight('font-bold'),
     fontSize('text-sm'),
     lineHeight('leading-5'),
-    transitionProperty('transition-transform'),
-    translate(animate ? '-translate-y-full' : undefined)
+    height(animate ? 'h-0' : undefined),
+    transitionProperty('transition-height', 'transition-padding'),
```

Comment:
Better do `transition-all`, take a look here: https://developer.mozilla.org/en-US/docs/Web/CSS/transition-property

Reply:
When you specify multiple transitions from tailwind, you specify many things like timing-function, duration: https://tailwindcss.com/docs/transition-property, they will just exist as dead code in CSS

Reply:
Got it, thank you